timemachine/cpp/src/bd_exchange_move.cu

```diff
@@ -58,8 +65,8 @@ BDExchangeMove<RealType>::BDExchangeMove(
     curandErrchk(curandCreateGenerator(&cr_rng_, CURAND_RNG_PSEUDO_DEFAULT));
     curandErrchk(curandSetPseudoRandomGeneratorSeed(cr_rng_, seed));
 
-    // Set the offset to the length of the random vector to ensure noise is triggered on first step
-    noise_offset_ = d_quaternions_.length;
+    // Set the batch size of random values to ensure noise is triggered on first step
+    noise_offset_ = RANDOM_BATCH_SIZE;
```

Comment:
for the BD Exchange Move, `noise_offset_` is only used a counter in the `move()` method, does it need to be a member? (no setters/getters on this either)